### ISSUE TITLE
Is mumoshu/aws-secret-operator:latest Docker image not latest?

### ISSUE THREAD

**Issue by tsub** [mumoshu/aws-secret-operator#1]
I installed according to [Installation](https://github.com/mumoshu/aws-secret-operator/tree/ab656595223cf686ea24733cc5cc057f145366e1#installation) in `README.md`.

But Pod was created instead of Secret.

```
$ kubectl get pods
NAME                                   READY   STATUS    RESTARTS   AGE
aws-secret-operator-8446b99787-9hw8z   1/1     Running   0          1m
example-pod                            1/1     Running   0          8s

$ kubectl get secrets
NAME                              TYPE                                  DATA   AGE
aws-secret-operator-token-v47zx   kubernetes.io/service-account-token   3      1h
default-token-fcjw2               kubernetes.io/service-account-token   3      1h
```

It was written `Creating a new Pod` in logs.

```
{"level":"info","ts":1549256976.6739526,"logger":"controller_awssecret","caller":"awssecret/awssecret_controller.go:114","msg":"Creating a new Pod","Request.Namespace":"default","Request.Name":"example","Pod.Namespace":"default","Pod.Name":"example-pod"}
```

However it was written `Creating a new Secret` in source codes.

https://github.com/mumoshu/aws-secret-operator/blob/ab656595223cf686ea24733cc5cc057f145366e1/pkg/controller/awssecret/awssecret_controller.go#L121

(The same is true for the first implemented commit https://github.com/mumoshu/aws-secret-operator/commit/dd2b3c8056fb4870bbd94e6d20db8cf39c07fca6)

## Questions

* Is `mumoshu/aws-secret-operator:latest` Docker image not latest? Should I use `:canary` tag?
* Should I build a Docker image myself? Just as written below

    https://github.com/mumoshu/aws-secret-operator/blob/ab656595223cf686ea24733cc5cc057f145366e1/deploy/operator.yaml#L18-L19

**Comment by zot24** [comment]
In the Makefile to publish `canary` its the tag being used so I'm assuming that's the latest version of the image to use

https://github.com/mumoshu/aws-secret-operator/blob/ab656595223cf686ea24733cc5cc057f145366e1/Makefile#L1

**Comment by tsub** [comment]
This issue solved in 57b274c
@mumoshu Thanks a lot!

**Comment by mumoshu** [comment]
@tsub Ah, sorry for not responding here! And thanks for your confirmation ☺️
